### ISSUE TITLE
Anchor links highlight wrong heading in TOC nav

### ISSUE THREAD

**Issue by PhilReedData** [ELIXIR-Belgium/elixir-toolkit-theme#291]
In my test SEEK docs site, I have some anchor links in the sidebar navigation, for example, from the [homepage](https://docs-testing.seek4science.org/help/user-guide/), expand the 'Assets' sidebar menu and click '[SOPs](https://docs-testing.seek4science.org/help/user-guide/general-attributes#sops)'. On the next page, the right-side TOC menu has 'Publications' highlighted, which is a different heading to the one selected. The address bar shows the correct URL (in this case, # sops).  
![image](https://github.com/user-attachments/assets/cbe33f43-1bf4-45c1-b152-610340440a06)

It works fine with 'Activities' then 'Events'. It does not work with 'General attributes and links' then 'Sharing', which points instead to 'Publishing' on the TOC most of the time. It works fine if you're already on the parent page.
![image](https://github.com/user-attachments/assets/26fb620c-97bc-4883-b023-a1b563bb8b41)

(I am trying to reduce the amount of these anchor links and duplicated links.)

I think this is a bug in the recent TOC code. I'm using ETT remote version 3.2.0.

**Comment by bedroesb** [comment]
To answer specific to your question. I would not want to add official support for anchors in the sidebar. But we can reshape the issue to make sure when people follow from scratch the link: https://docs-testing.seek4science.org/help/user-guide/general-attributes#sops, the correct TOC item is highlighted

**Comment by bedroesb** [comment]
Also the fact that the sidebar does not highlight anchor links correctly should be a giveaway that the theme is not made for it!

**Comment by PhilReedData** [comment]
Hi Bert, thanks for this. I've now found a sensible way to remove all the anchor links from this sidebar. For example, the top Samples page now has the 'Browsing samples' links in the page itself, not the sidebar.
The test server will be auto updated with my latest code overnight tonight. 
<img width="1619" alt="image" src="https://github.com/user-attachments/assets/6fb53686-d0ed-4630-a750-35628e636812">
Above: Samples. Below: Attributes.
<img width="1619" alt="image" src="https://github.com/user-attachments/assets/e1268c68-2ee0-48f5-a340-7eb11537629b">
That should stop most of the surprise scrolling. I still have two pages that appear twice in the menu, but that has nothing to do with the template!

**Comment by bedroesb** [comment]
Looks a lot better! I just notice there is a child item with the same name is the parent item, "Samples". You can put pages behind parent collapsables too if needed.

**Comment by PhilReedData** [comment]
Indeed. I wasn't sure about this. All of the other parents / sections do not have their own page, they are just collections of pages. Samples is different as there is a page called Samples. To be consistent with the other sections, I kept the two items labelled Samples but that is a bit confusing in a different way, two links with the same name. Perhaps there is a better label for one of them.

**Comment by PhilReedData** [comment]
'Samples' and 'Samples introduction' is better. 
<img width="1619" alt="image" src="https://github.com/user-attachments/assets/9611ab9c-635f-4513-9a6c-27f2e659c774">
